src/fault_injector/fault_types/train_fault.py

```diff
@@ -0,0 +1,26 @@
+from src.component import Component
+from src.fault_injector.fault_types.fault import Fault
+
+
+class TrainFault(Fault):
+    """A fault affecting trains."""
+
+    @classmethod
+    def from_json(cls, json_object: str) -> "Fault":
```

Comment:
```suggestion
    def from_json(cls, json_object: str) -> Fault:
```
There should be no need to specifiy the return type as a string